src/redfish/rest/v1.py

```diff
@@ -951,11 +951,11 @@ def _rest_request(self, path, method='GET', args=None, body=None,
 
                     try:
                         LOGGER.debug('HTTP RESPONSE for %s:\nCode: %s\n\nHeaders:\n' \
-                                 '%s\nBody Response of %s: %s\n'%\
+                                 '%s\nBody Response of %s:\n%s\n'%\
                                  (restresp.request.path,
                                 str(restresp._http_response.status_code)+ ' ' + \
                                 restresp._http_response.reason,
-                                headerstr, restresp.request.path, restresp.read))
+                                headerstr, restresp.request.path, json.dumps(json.loads(restresp.read), indent=4)))

```

Comment:
Could we have it dynamically check if the response body is JSON-encoded and fall back on the old method if it's not JSON?

Reply:
I've updated, do you think is this ok?

Reply:
Looks good to me! Thanks!